### DISCUSSION TITLE
Ignore Fix

### DISCUSSION THREAD

**Discussion by real-yfprojects** [astral-sh/ruff#4980] [Q&A] [answered]
I would like to have all fixes enabled by default (which is the default) but disable the fixes for a few specific rules. AFAIK this is only possible by specifying all other rules in `fixable` which would be very tedious. Is there another way?

**Comment by MichaReiser** [comment, marked as answer]
Hey. 

Yeah, that's annoying. Would `unfixable` work for you (from the CLI documentation but is also available in the configuration):

> --unfixable <RULE_CODE>
          List of rule codes to treat as ineligible for autofix. Only applicable when autofix itself is enabled (e.g., via `--fix`)

**Reply by real-yfprojects** [reply]
Thanks, that is exactly what I was looking for. I have to admit alphabetic ordering has it advantages but it doesn't help finding options you didn't know of before. I was trying to find the option grouped with `fixable` since that would make sense logically.
